### PR TITLE
test: update integration test settings

### PR DESCRIPTION
**Pull request by ssmmnn11**
modify some integration test settings:

awkward batch size
lr and scheduler to have a realistic learning rate
less processor layers for faster tests

### CODE REVIEW

**Comment by ssmmnn11** [comment]
Yes, we would need to update the checkpoint (load existing checkpoint) - from 16 to 2 layers.

File: training/tests/integration/config/testing_modifications.yaml [hunk 1]
```diff
@@ -4,11 +4,11 @@ dataloader:
...
   batch_size:
-    training: 2
-    validation: 2
+    training: 3
+    validation: 3
   limit_batches:
-    training: 2
-    validation: 2
+    training: 3
+    validation: 3
```
Comment (VeraChristina) [inline training/tests/integration/config/testing_modifications.yaml:11]: I think it's fine to keep the limit to two batches with `batch_size=3`

File: training/tests/integration/config/testing_modifications.yaml [hunk 2]
```diff
@@ -38,6 +38,12 @@ graph:
 
 model:
   num_channels: 16
+  processor:
+    num_layers: 2
```
Comment (VeraChristina) [inline training/tests/integration/config/testing_modifications.yaml:42]: wondering if reducing layers will make the tests much faster given that we run very short cycles on reduced data and a lot of time is overhead+plotting -- but that's not to say we shouldn't change this

File: training/tests/integration/config/testing_modifications.yaml (hunk 1 above)
Comment (VeraChristina) [reply training/tests/integration/config/testing_modifications.yaml:11]: I think this is why the test_restart_training test fails

**Review by VeraChristina — COMMENTED** [review]
Thanks for adding this! Two minor comments below. In the run linked above there seem to be two tests that are failing (in addition to the ones where we would need to update the checkpoints): test_restart_training, and test_training_cycle_diffusion

File: training/tests/integration/config/testing_modifications.yaml (hunk 1 above)
Comment (ssmmnn11) [reply training/tests/integration/config/testing_modifications.yaml:11]: I wanted to go to three because then we have a bit more of a "realistic" learning rate scenario. together with warmup etc.

File: training/tests/integration/config/testing_modifications.yaml (hunk 2 above)
Comment (ssmmnn11) [reply training/tests/integration/config/testing_modifications.yaml:42]: I think on the CPU yes, but increasing the batch size and the number of samples brings us approx. to where we were before I think. Still a more realistic test.

File: training/tests/integration/config/testing_modifications.yaml (hunk 1 above)
Comment (VeraChristina) [reply training/tests/integration/config/testing_modifications.yaml:11]: Makes sense! We might need to tweak the settings of the restart test but that seems fine
